### PR TITLE
feat(ci): add federated integ test to ci

### PR DESCRIPTION
**Pull request by uri200**
Signed-off-by: Oriol Batalla <obatalla@fb.com>

## Summary

This PR adds Federated Integ Test to CI. This task is a modification of l[te-integ-test.](http://github.com/magma/magma/blob/master/.github/workflows/lte-integ-test.yml#L3-L3)

Even this works there are possible enhancements to make it faster or more reliable:
1. Modify this job to run in a ubuntu box. That will allow us to install docker on the ubuntu box. This may requiere changes on the fab script and on Vagrant vms to make sure they work in both, linux and MacOS (remember the test needs to run on MacOs so developers can use it locally)

2. Modify s1ap federated integ test to run Orc8r inside the magma Vagrant machine. That may work but that would make a mess on the local scenario. Also this doesn't assure that the runner takes hours to run due to low ram memory (docker over vagrant over the runner) 

3. Run S1ap Federated integ test on a AWS node that can support Virtualization

Enhancements
- Feg and Orc8r are build previously by other jobs. Those images are uploaded to magma core jfrog docker registry. Instead of building them again in this task, we could point the vm docker to the JFROG repo and grab the already built image from there.

## Test Plan

For this test we triggered the job to run Federated Integ test on PRs. This is triggered here https://github.com/magma/magma/pull/12633/files#diff-6a95f663e4dfd08ee0988d079f574d7c7782a84b3b3ba2459a8b4891e48ca944R15-R24 . But once this work, we can remove those lines and just execute this on merge. This code is just here so we are able to execute the test and troubleshoot it

image

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Comment by uri200** [comment]
macos does not support docker. It can not be installed properly on a runner

**Comment by maxhbr** [comment]
maybe use a minikube or similar as docker host. But it is not a no-brainer. I will think about that

File: .github/workflows/federated-integ-test.yml
Comment (maxhbr) [inline .github/workflows/federated-integ-test.yml]: this is currently a Draft?

Comment (uri200) [reply .github/workflows/federated-integ-test.yml]: We are testing some stuff on CI.

Let me add Work in progress tag

Comment (Neudrino) [reply .github/workflows/federated-integ-test.yml]: Hi @uri200,
if this is a draft, you could consider to convert it to a draft PR, to get it of peoples review list. :wink: 
grafik

File: .github/workflows/federated-integ-test.yml
Comment (uri200) [inline .github/workflows/federated-integ-test.yml]: this is commented out to avoid this to be run
This will be fixed  by 
https://github.com/magma/magma/pull/13012

**Review by emakeev — APPROVED** [review]
LGTM

**Review by koolzz — APPROVED** [review]

File: .github/workflows/federated-integ-test.yml
Comment (crasu) [inline .github/workflows/federated-integ-test.yml]: I guess this has to go before we merge this

File: .github/workflows/federated-integ-test.yml
Comment (crasu) [inline .github/workflows/federated-integ-test.yml:149]: Do we need the ls here?

**Review by crasu — APPROVED** [review]

**Review by ardzoht — APPROVED** [review]

File: .github/workflows/federated-integ-test.yml
Comment (uri200) [reply .github/workflows/federated-integ-test.yml]: yep, this si gone now

File: .github/workflows/federated-integ-test.yml
Comment (uri200) [reply .github/workflows/federated-integ-test.yml:149]: I am not totally sure. Per my understanding that was being used by `magma-ci`. So I guess that next step is to integrate it with magma-ci and use those summaries

Comment (crasu) [reply .github/workflows/federated-integ-test.yml:149]: ok fine

**Comment by crasu** [comment]
@uri200 I guess we can merge this now

**Comment by uri200** [comment]
> @uri200 I guess we can merge this now

Hey @crasu  for some reason it does not let me merge. Not sure if there is a missing approval ?

**Comment by uri200** [comment]
@Neudrino  can you approve this PR to see if then I can merge it?

**Review by tmdzk — APPROVED** [review]

**Pull request merged**
